components/style/parallel.rs

```diff
@@ -37,9 +38,11 @@ use traversal::{DomTraversal, PerLevelTraversalData, PreTraverseToken};
 ///
 /// Larger values will increase style sharing cache hits and general DOM locality
 /// at the expense of decreased opportunities for parallelism. The style sharing
-/// cache can hold 8 entries, but not all styles are shareable, so we set this
-/// value to 16. These values have not been measured and could potentially be
-/// tuned.
+/// cache can hold 31 entries, but not all styles are shareable, so we set this
+/// value to 16. The size of the cache has been measured to provide pretty good
```

Comment:
This comment doesn't make sense anymore. The whole point was that we originally had the work unit size be double the cache size. So document the reasoning of why we separated those (what we discussed on IRC a few days ago).